cli/signal.rs

```diff
@@ -11,7 +21,33 @@ pub fn kill(pid: i32, signo: i32) -> Result<(), OpError> {
 }
 
 #[cfg(not(unix))]
-pub fn kill(_pid: i32, _signal: i32) -> Result<(), OpError> {
-  // TODO: implement this for windows
+pub fn kill(pid: i32, signal: i32) -> Result<(), OpError> {
+  unsafe {
+    match signal {
+      9 | 15 | 2 => {
+        let handle = if pid == 0 {
+          GetCurrentProcess()
```

Comment:
On other platforms, kill(0) does not kill the current process, but rather the current process group.
Closes equivalent on windows would be to terminate the current job object, but let's not implement that now.
Preferrably just error with "not supported".